mapdata/WarcraftLegacies/UpgradeData/Skin/909324370.json

```
@@ -43,7 +43,7 @@
       "Level": 1,
       "Id": 828536167,
       "Type": 3,
-      "Value": "Improves Obsidian Drakes by causing their attacks to deal \u003CA0DS,DataA1\u003E additional damage on the first attack, twice as much on the second attack, three times as much on the third attack, etc.|n|nIf a unit dies while under this effect, it is incinerated, causing up to \u003CA0DS,DataB1\u003E damage to all nearby hostile units."
+      "Value": "Improves Obsidian Drakes by causing their attacks to deal cause fire to cling to a unit. If a unit dies while under this effect, it is incinerated, causing up to \u003CA0DS,DataB1\u003E damage to all nearby hostile units."
```

```suggestion
      "Value": "Improves Obsidian Drakes by causing their attacks to cause fire to cling to a unit. If a unit dies while under this effect, it is incinerated, causing up to \u003CA0DS,DataB1\u003E damage to all nearby hostile units."
```
Also FYI these Incinerate tooltip changes will conflict with the similar fix I did in dev, but that's OK - I'll just keep dev for those.